Employment Policies/Working Remotely.md

```diff
@@ -1,11 +1,91 @@
 # Working Remotely
 
-As a small team, things move very quickly and decisions are made or reversed as new information helps guide us. This makes it really hard to keep up with company progress for anyone who isn’t in the office participating in all of the conversations that are going on. As a result, we put a high premium on physical colocation, even though this limits the geography that we can recruit from.
+## Scope
 
-Every employee is an important contributor to the decisions we make and the work we do as a team, so we expect full-time employees to be at the Clef office between 45 and 50 hours a week, and to be punctual for team meetings. If employees have a consideration which requires them to be out of the office, they should get approval from the founder they report to as far in advance as possible, and at least a full day before the absence.
+This policy is written in the context of our current company setup, which is that the majority of employees work in the same office for the most of the work week. Specifically, this document addresses working from home on a regular basis and/or working remotely for up to one week out of any given month. It does not address:
 
-Employees with chronic or terminal illnesses should talk with the founder they report to about their needs for remote work, flexible time, and other support.
+* A fully remote employee
+* An employee who works from the office 1wk/month or is remote for the majority of their time
+* A team that is distributed
 
-As we grow, we hope to add the capability for employees to work from home or work remotely more consistently, but, at our current size, the in-person interactions in the office are too critical to pass up.
+If and when the current company setup changes, we should address the above situations. For the purpose of this document, we interpret remote work as as a co-located employee of Clef working from home or from an irregular-place. 
 
-When we reach 10 employees, we will review this policy to see if our capabilities to support remote work have expanded.
+## Approach
+
+Fundamentally, working remotely is a privilege that comes with both benefits and costs. We should treat it as such.
```

Comment:
Clarity: how about "We treat working remotely as a privilege that comes with both benefits and costs."